BGD_2020_Markets_Covid/scripts/functions/GIS_Project_Functions.R

```diff
@@ -0,0 +1,27 @@
+to46N<-function(spatial_object){
+  spatial_object_utm<-spTransform(spatial_object, CRS("+proj=utm +zone=46 +datum=WGS84 +units=m +no_defs +ellps=WGS84 +towgs84=0,0,0"))
+  return(spatial_object_utm)
+}
+towgs84<-function(spatial_object){
+  spatial_object_wgs<-spTransform(spatial_object, CRS("+proj=longlat +datum=WGS84 +no_defs +ellps=WGS84 +towgs84=0,0,0"))
+  return(spatial_object_wgs)
+}
+
+
+spatial_join<-function(point,poly){
+  if (class(point)=="SpatialPoints"){
+    df_over<-over(point,poly)
```

Comment:
you are using functions from the sp package to make new functions.  I recommend forgetting about the sp package and only use sf package